docs/field-fft.ipynb

```diff
@@ -0,0 +1,668 @@
+      "text/plain": [
+       "<Figure size 800x600 with 2 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "field.sel(\"x\").mpl()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8d2f8678",
+   "metadata": {},
+   "source": [
+    "### FFT Methods\n",
+    "\n",
+    "The `discretisedfield` package provides four different methods for performing Fast Fourier Transform (FFT) operations on a `discretisedfield.Field` object:\n",
+    "\n",
+    "1. `fftn`: This is the \"full\" FFT method which computes the n-dimensional FFT. It returns the complex-valued FFT result that has the same shape as the input field. It's used when the full frequency spectrum is needed.\n",
```

Comment:
"Frequency" sounds a bit like we would start from time, not space. Can we make more clear that it is "frequencies" in k space?